scipy/stats/_distribution_infrastructure.py

```diff
@@ -1307,9 +1355,9 @@ def _generate_example(dist_family):
         instantiation = f"{name}()"
         X = dist
 
-    p = 0.32
-    x = round(X.icdf(p), 2)
-    y = round(X.icdf(2 * p), 2)
+    prob = 0.32
```

Comment:
I thought this was interfering with the shape parameter `p`, but in retrospect I don't think this is necessary.